index.rst

```diff
@@ -19,6 +19,17 @@ Each of the four Jina products has the following documentation types.
 
 ------------
 
+Jina Products
+**********************
+
+
+- **Jina Core**: is the open-source neural search framework, it allows you to build search systems for any kind of data.
+- **Jina Dashboard**: is a low-code monitoring and management environment for Jina. With the Dashboard you can build Jina Flows, manage them from a central location and get insights into the health of the Flows.
+- **Jina Hub**: is a centralized registry, where you can share and discover custom and community driven Jina Pods and Apps tailored to specific use cases of neural search.
```

Comment:
we dont offer Apps yet so maybe take that out

Reply:
Are Apps something different? I would assume they are just pods.

Reply:
But yeah I can remove it

Reply:
its something else we have plans for in the future i think